pkg/daemon/daemon.go

```diff
@@ -82,6 +85,10 @@ type Daemon struct {
 	nodeLister       corev1lister.NodeLister
 	nodeListerSynced cache.InformerSynced
 
+	// imageLister is for retrieving our image details for application
+	imageLister       imagelistersv1.ImageLister
```

Comment:
This doesn't need to be a lister, but I didn't have time to take it back out. I don't need to watch/list images, I really just need to get the one image object for the SHA I've been given.